src/main/java/com/hotels/bdp/circustrain/bigquery/partition/PartitionColumnFormatter.java

```diff
@@ -34,20 +34,16 @@ public static String formatColumns(List<FieldSchema> columns) {
       String name = col.getName().trim();
       String type = col.getType().toLowerCase().trim();
 
-      if (type.equals("timestamp")) {
-        formattedColumns.add(formatTimestampColumn(name));
-      } else {
-        formattedColumns.add(name);
-      }
+      formattedColumns.add(name);
     }
 
     return joiner.join(formattedColumns);
   }
 
-  // format the timestamp returned by BigQuery to include the timestamp without timezone
-  // if timestamp contains timezone, Hive converts that value to null
+  // format the timestamp returned by BigQuery to transform it into a long
+  // avro represents timestamps as long
   private static String formatTimestampColumn(String name) {
```

Comment:
This method is no longer used and can be removed right?

Reply:
Oh yeah, that's true!